t/Clojars.t

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+use Test::More;
+use DDG::Test::Spice;
+
+spice is_cached => 1;
+
+ddg_spice_test(
+    [qw( DDG::Spice::Clojars)],
+    # At a minimum, be sure to include tests for all:
```

Comment:
nit: you can remove this comment, and the one below. They're just for you to read and see